### ISSUE TITLE
warning when registering python 3.11 toolchain

### ISSUE THREAD

**Issue by pdeva** [bazelbuild/rules_python#1405]
# 🐞 bug report

### Affected Rule

 The issue is caused by the rule:  `python.toolchain`

### Is this a regression?

 Dunno. We started noticing this since latest version and switching to bzlmod.

### Description

Getting this warning message when registering python 3.11 toolchain. I suspect its cause rules_python already [configures its own toolchain](https://github.com/bazelbuild/rules_python/blob/main/MODULE.bazel#L46).  

Is there no way to register a python 3.11 toolchain without the warning message? 

## 🔬 Minimal Reproduction

```python
bazel_dep(name = "rules_python", version = "0.25.0")
python = use_extension("@rules_python//python/extensions:python.bzl", "python")
python.toolchain(
    is_default = True,
    configure_coverage_tool = True,
    ignore_root_user_error = True,
    python_version = "3.11",
)
```

## 🔥 Exception or Error

<pre><code>

DEBUG: /opt/github-workspace/.cache/bazel/_bazel_runner/edf2f0ebce57525bd69d2f6271ecfed5/external/rules_python~0.25.0/python/extensions/python.bzl:44:10: 
     WARNING: Ignoring toolchain 'python_3_11' from module 'rules_python': 
          Toolchain 'python_3_11' from module 'astradot_monorepo' already registered Python version 3.11 and has precedence
</code></pre>

## 🌍  Your Environment

**Operating System:**

<pre>
  <code>
Debian 12 arm64

Distributor ID:	Debian
Description:	Debian GNU/Linux 12 (bookworm)
Release:	12
Codename:	bookworm
  </code>
</pre>

**Output of `bazel version`:**

<pre>
  <code>
Bazelisk version: development
Starting local Bazel server and connecting to it...
Build label: 6.3.2
Build target: bazel-out/aarch64-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Tue Aug 8 15:48:06 2023 (1691509686)
Build timestamp: 1691509686
Build timestamp as int: 1691509686
  </code>
</pre>

**Rules_python version:**

<pre>
  <code>
0.25.0
  </code>
</pre>

**Anything else relevant?**

**Comment by chrislovecnm** [comment]
The warning is actually that, a warning. This is expected behavior. You cannot register the same python version twice, so we only register one. I’m gonna close this. I’m happy to reopen if you have questions or are concerned about what happens.
